tensorboard/webapp/metrics/store/metrics_reducers_test.ts

```diff
@@ -1232,6 +1232,21 @@ describe('metrics reducers', () => {
       expect(thirdState.settings.hideEmptyCards).toBe(false);
       expect(thirdState.settingOverrides.hideEmptyCards).toBe(false);
     });
+
+    it('changes savingPinsEnabled on metricsEnableSavingPinsToggled', () => {
```

Comment:
Since this tests a "toggle", could we have at least two test cases minimum here? One changing from true -> false, another from false -> true (parameterized tests may help - please see other comment)

Reply:
I added some states to make it true -> false -> true.

Reply:
nit: interleaving act and assert steps is explicitly forbidden by our style guide and I think we should follow it unless there's a very strong reason not to. Can we just create separate tests for true->false and false->true?

Reply:
Applied. Thanks for reminding me! I just followed this interleaving act and assert steps because it was commonly used in this test codebase

Reply:
Thanks for the cleanup! My readability meta-mentor linked me to this when I asked if consistency is important: https://peps.python.org/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

My stance now is `the right thing` > `consistency`, unless inconsistency causes big problems